lxd/db/images.go

```diff
@@ -29,6 +29,7 @@ import (
 //go:generate mapper stmt -p db -e image objects-by-Project-and-Fingerprint-and-Public
 //go:generate mapper stmt -p db -e image objects-by-Fingerprint
 //go:generate mapper stmt -p db -e image objects-by-Cached
+//go:generate mapper stmt -p db -e image objects-by-AutoUpdate
```

Comment:
Should split this commit in half, one half for this comment + result of the code generator and then a second one for the manually added function.